### PR TITLE
feat: additional parameter files

### PR DESCRIPTION
**Pull request by bondfelix**
## Purpose

I would like to enable the property `serviceparameterfile` on `sapbtp` as well on `kymaruntime` objects.

E.g. to enable this

```json
{
    "category": "ENVIRONMENT",
    "name": "kymaruntime",
    "instancename": "ccircularity-runtime",
    "plan": "aws",
    "amount": 1,
    "serviceparameterfile": "$BTPSA_PARAMETERS_SERVICE_KYMA"
}
```

or that

```json
{
    "category": "SERVICE",
    "repeatstatusrequest": 60,
    "repeatstatustimeout": 3600,
    "name": "hana-cloud",
    "plan": "hana",
    "targetenvironment": "sapbtp",
    "instancename": "ccircularity-db",
    "serviceparameterfile": "$BTPSA_PARAMETERS_SERVICE_HANA"
}
```

## Does the PR solve an issue

```
[ ] Yes - Please add issue number
[X] No
```

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type

What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

See configuration examples above.

## What to Check

Verify that the following are valid

* ...

### CODE REVIEW

File: libs/python/btp_cli.py [hunk 1]
```diff
@@ -724,6 +724,13 @@ def create_environments(self):
                         self.create_and_assign_quota_plan(environment)
 
                 elif environment.name == "kymaruntime":
+                    # Support load via dynamic parameter file as well
+                    if environment.parameters is None:
+                        # try via serviceparameter file
+                        command = f"cat \"{environment.serviceparameterfile}\""
+                        message = "Read out environment parameter fie"
```
Comment (lechnerc77) [inline libs/python/btp_cli.py:731]: Typo: `fie` -> `file`

File: libs/python/btp_cli.py [hunk 2]
```diff
@@ -724,6 +724,13 @@ def create_environments(self):
                         self.create_and_assign_quota_plan(environment)
 
                 elif environment.name == "kymaruntime":
+                    # Support load via dynamic parameter file as well
```
Comment (lechnerc77) [inline libs/python/btp_cli.py:727]: Format seems not according to `black` rules,

Can you please run `black .\libs\python\* -t py310`  
See <https://github.com/psf/black>

**Review by lechnerc77 — CHANGES_REQUESTED** [review]
Minor changes required

**Comment by lechnerc77** [comment]
Hi @bondfelix ,

thanks a lot for your contribution to the project! Highly appreciated. I have just two small remarks before we can merge your change into the `dev` branch.

**Review by lechnerc77 — APPROVED** [review]
LGTM

**Comment by rui1610** [comment]
LGTM, too. Thanks a lot @bondfelix for your contribution!

**Comment by lechnerc77** [comment]
Available in dev; will be merged to stable/main with next release

**Pull request merged**